src/Entities/QuestionInstance.php

```diff
@@ -49,4 +54,20 @@ public function setText(string $text)
     {
         $this->text = $text;
     }
+
+    /**
+     * @ORM
+     */
```

Comment:
why do you need this annotation here?

Reply:
This is for hydrator, on extraction from Database it will know to call this method